### DISCUSSION TITLE
Discord

### DISCUSSION THREAD

**Discussion by ghost** [zenkovich/o2#35] [General]
want to make a discord channel for community?
I think this is a good first step for the community to have discord, sharing ideas and showing off projects on discord can increase motivation to make games.
I'm not a game engine developer, but I always want to know how game engines work, and your game engine has an interesting interface, thank you for continuing this game engine project.

**Comment by anzenkovich** [comment]
I used to think that there is no need to create a discord or something like that until there is a beta version. But now you gave me an idea, why not? Moreover, someone clearly asked about it. So welcome - https://discord.gg/6hstsSjf
